src/Ledger/Utxo/Properties.lagda

```diff
@@ -39,23 +40,35 @@ open Tx
 open Equivalence
 open Properties
 
-import Data.Nat.Tactic.RingSolver as ℕ
 open Tactic.EquationalReasoning.≡-Reasoning {A = ℕ} (solve-macro (quoteTerm +-0-monoid))
 
 instance
   _ = TokenAlgebra.Value-CommutativeMonoid tokenAlgebra
   _ = +-0-monoid
 
+record Add (A : Set) : Set where
+  infixl 7 _+_
+  field _+_ : A → A → A
+
+open Add ⦃ ... ⦄ public
+
+instance
+  addInt : Add ℤ
+  addInt = record { _+_ = _+ℤ_ }
+
+  addNat : Add ℕ
+  addNat = record { _+_ = _+ℕ_ }
+
+  addCommMon : Add (TokenAlgebra.Value tokenAlgebra)
+  addCommMon = record { _+_ = _+ᵛ_ }
+
```

Comment:
This is just one way to add a very basic "type-class"-like feature by which Agda knows which `_+_` symbol we mean.  Later we may want to add some more sophisticated instance records, such as Monoids, or Commutative Monoids, or Groups, or Symmetric Monoidal Categories, or... but I just wanted to get something simple done and working and avoid getting bogged down by unnecessarily complex or fancy type classes.